dist/router.es5.js

```diff
@@ -148,6 +148,7 @@ function routerViewPortDirective($animate, $compile, $controller, $templateReque
     }, viewPortName);
   }
 }
+routerViewPortDirective.$inject = ["$animate", "$compile", "$controller", "$templateRequest", "$rootScope", "$location", "$componentLoader", "$router"];
```

Comment:
I haven't really looked into the new router (yet), so I probably miss something, but why are those `$inject`s necessary if we are passing it through `ngAnnotate` ?